pom.xml

```diff
@@ -408,7 +408,7 @@
             <dependency>
                 <groupId>io.prestosql.hive</groupId>
                 <artifactId>hive-apache</artifactId>
-                <version>1.2.2-3</version>
+                <version>2.3.4-1-SNAPSHOT</version>
```

Comment:
how is that possible that automation is green? Have you deployed SNAPSHOT?

Reply:
Yes, I deployed the snapshot so that I could test Travis before releasing it